### PR TITLE
Added linter config for non-checked error block.

### PR DESCRIPTION
**Pull request by yusufpapurcu**
I added inline linter config for specific lines. This configs disable error checks for added lines.

### CODE REVIEW

**Comment by mreiferson** [comment]
Thanks for the PR. Curious what linter this is for? Here's the output I get for `golint`:

```
$ golint
diskqueue.go:19:6: exported type LogLevel should have comment or be unexported
diskqueue.go:22:2: exported const DEBUG should have comment (or a comment on this block) or be unexported
diskqueue.go:29:6: exported type AppLogFunc should have comment or be unexported
diskqueue.go:47:6: exported type Interface should have comment or be unexported
diskqueue.go:407:2: should replace d.depth += 1 with d.depth++
diskqueue.go:560:2: should replace d.depth -= 1 with d.depth--
```

File: diskqueue.go [hunk 1]
```diff
@@ -548,7 +548,7 @@ func (d *diskQueue) checkTailCorruption(depth int64) {
 				d.name, d.readPos, d.writePos)
 		}
 
-		d.skipToNextRWFile()
+		d.skipToNextRWFile() //nolint:errcheck
```
Comment (yusufpapurcu) [inline diskqueue.go:551]: golangci-lint warning in this line.

**Comment by ploxiln** [comment]
the particular linter that flags these lines is https://github.com/kisielk/errcheck

but this ignore syntax is for "golangci-lint", which runs many different linters built into a single binary/process. It's kinda like how flake8 runs pyflakes - pyflakes doesn't have an ignore feature, but flake8 adds one on top.

https://golangci-lint.run/usage/linters/

**Comment by yusufpapurcu** [comment]
Oh I can understand. Thanks for information. I'm closing this pr.

**Pull request closed without merging**